types/requests.ts

```diff
@@ -245,3 +245,81 @@ export interface CodeExecutionTool {
    */
   codeExecution: {};
 }
+
+/**
+ * Request message for generating image.
+ * @public
+ */
+export interface ImageGenerationRequest {
+  /**
+   * Text prompt for the image.
+   */
+  prompt: string;
+  /**
+   * A description of what you want to omit in the generated images.
+   */
+  negativePrompt?: string;
+  /**
+   * Number of images to generate. Range: 1..8.
+   */
+  numberOfImages?: number;
+  /**
+   * Width of the image. One of the Width/Height sizes must be 256 or 1024.
+   */
+  width?: number;
+  /**
+   * Height of the image. One of the Width/Height sizes must be 256 or 1024.
+   */
+  height?: number;
```

Comment:
When I specify `width` and `height` to `256`, I get back an image that is `1024x1024`. I'm not sure whether this is a backend issue or not.